pset-functions.js

```diff
@@ -207,7 +280,18 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 5:')
 
 // Add your code below this line
-
+const word = function(letter) {
```

Comment:
@deedeemedina nice job getting the solution working. Using a nested `for` loop works well here. One of the drawbacks of using nesting loops is that it can lead to complex code that is hard to debug. 

[See the solution](https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L300) for an alternative approach to this problem that does